### ISSUE TITLE
Keyword search change request

### ISSUE THREAD

**Issue by Edje3001** [cgeo/cgeo#10271]
Currently, when you select search from the main menu and then fill in keywords for the keyword search, the only returned caches are the ones where the title is beginning with the requested keyword(s). It would be a great feature if also caches which contain the requested keyword in the title are returned.

**Comment by wolverine007** [comment]
This function is already working, so no implementation is needed.

**Comment by fm-sys** [comment]
Indeed, the original issue exists, but I guess this is not solvable from our side...

![04042021092107_copy_432x560](https://user-images.githubusercontent.com/64581222/113502255-683ea380-952b-11eb-8885-5ca8c896172a.jpg)

**Comment by Edje3001** [comment]
Sorry to say: please test! This is not working in the latest beta release
and releases before.

In the search function, when searching for a keyword, only caches are
returned that START with the keyword in the name, no caches are returned
with the keyword somewhere else in the name of the cache.

Op zo 4 apr. 2021 07:10 schreef wolverine007 ***@***.***>:

**Comment by Lineflyer** [comment]
Thats the way the search works on geocaching.com

**Comment by wolverine007** [comment]
> Sorry to say: please test! This is not working in the latest beta release
> and releases before.
> 
> In the search function, when searching for a keyword, only caches are
> returned that START with the keyword in the name, no caches are returned
> with the keyword somewhere else in the name of the cache.
> 
> Op zo 4 apr. 2021 07:10 schreef wolverine007 ***@***.***>:
> 
> > This function is already working, so no implementation is needed.
> > .
> >
> 

It is working but only for the list which is generated as a pop up while you are typing the search - when you click on the magnifier and the search window is opening, only the caches which are starting with your search are shown.

Maybe the popup is doing a different search above the locally stored caches?
